lib/compiler/build_runner.zig

```diff
@@ -692,7 +692,12 @@ fn printStepStatus(
         },
         .skipped, .skipped_oom => |skip| {
             try ttyconf.setColor(stderr, .yellow);
-            try stderr.writeAll(" skipped");
+            try stderr.writeAll(" skipped: ");
+            if (s.result_skip_reason) |reason| {
+                try stderr.writeAll(reason);
+            } else {
+                try stderr.writeAll("Unspecified");
+            }
```

Comment:
Could this just not specify anything when no reason is given?

Reply:
That wouldn't be a problem. Thinking about it that makes more sense, I'll change it